bigquery_etl/docs/mozdata/generate_mozdata_docs.py

```diff
@@ -0,0 +1,65 @@
+import os
+import yaml
+from pathlib import Path
+from jinja2 import Environment, FileSystemLoader
+
+VIEW_FILE = "view.sql"
+METADATA_FILE = "metadata.yaml"
+NON_USER_FACING_DATASET_SUFFIXES = (
+    "_derived",
+    "_external",
+    "_bi",
+    "_restricted",
+)
+SOURCE_URL = "https://github.com/mozilla/bigquery-etl/blob/master"
+
+
+def generate_mozdata_docs(out_dir, project_dir):
```

Comment:
Is there something that makes this specific to mozdata, instead of being a general purpose function?

Reply:
yeah how we generate docs for mozfun and mozdata is different (e.g. you need to load examples for mozfun, when it's not needed for mozdata). Also the main function was getting quite long, so I thought it'd be better to abstract this out.

Reply:
It seems like this function would work for something like `moz-fx-data-marketing-prod`, if we wanted to generate docs there too.

Reply:
Good point! I changed the function name to `generate_derived_dataset_docs`. Hope that's better :)